README.md

```diff
@@ -150,7 +151,7 @@ If you used `make` to deploy, then run the following to uninstall Gatekeeper:
 
 If you used `helm` to deploy, then run the following to uninstall Gatekeeper:
 ```sh
-helm delete <release name> --purge
+helm delete <release name>
```

Comment:
Should we also add some verbiage about having to delete the custom resource definitions explicitly with kubectl?

Reply:
You mean for cleaning up CRDs?

Reply:
added instructions for crd uninstall